src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -134,10 +136,16 @@ public void run(@NotNull ProgressIndicator progressIndicator) {
 
                 // Set the progress bar percentage and text
                 codeSyncProgressIndicator.setMileStone(InitRepoMilestones.START);
-                setupCodeSyncRepo(project, repoPath, repoName, codeSyncProgressIndicator, skipSyncPrompt, isSyncingBranch);
+
+                try {
+                    setupCodeSyncRepo(project, repoPath, repoName, codeSyncProgressIndicator, skipSyncPrompt, isSyncingBranch);
```

Comment:
@syedgulahmed What exception are you catching here?
I think we decided to check output of `validateAccessToken` function? right? as it already handles exceptions internally and not raise anything.